packages/webapp/src/components/ui/ActionBar/index.tsx

```diff
@@ -27,6 +36,9 @@ export interface ActionBarProps {
 export const ActionBar: StandardFC<ActionBarProps> = memo(function ActionBar({ title }) {
 	const { isMD } = useWindowSize()
 	const { c } = useTranslation()
+	const isDurableCacheEnabled = Boolean(config.features.durableCache.enabled)
```

Comment:
Don't we have a config utils file to store those kind of Boolean test?
If not don't worry about it.